.github/workflows/tests.yml

```diff
@@ -60,8 +60,13 @@ jobs:
               name: context.repo.repo,
               label: 'CI-long-timeout'
             }
-            const { repository: { pullRequests: { totalCount } } }  = await github.graphql(query, variables)
-            core.setOutput('count', totalCount)
+            const response = await github.graphql(query, variables)
+            if (response.errors) {
+              // The GitHub API query errored, so fail open and assume 0 PRs.
+              core.setOutput('count', 0)
+            } else {
+              core.setOutput('count', response.data.repository.pullRequests.totalCount)
+            }
```

Comment:
Maybe more idiomatic to do `try ... catch`? (I don't do JavaScript though, so this could be a lie.)

Alternatively, I think you can move this whole snippet inside the `if labels.include(...)` block of the `setup_tests` job so that we only query the API when needed. (I realised this belatedly but thought it could wait till I got back to it...)

Reply:
It's 11pm, I barely write idiomatic JS during working hours. 😅

Yeah, I wondered why this was separate and running for everything, but assumed you had a reason for it and steamed ahead to fix the immediate problem.

Reply:
Well the reason was that workflow + JavaScript syntax is hard and I was too gassed by the time I got to something that worked to be thinking straight 😅

Reply:
Now I've moved it I have to add the label to this PR to see if it works. ⏳

Reply:
Wait, I think all the formula labelling conditional code only runs if a formula has changed? So I need to do more in this PR to test it...

I'm beginning to think my original solution was simpler, but I can understand and support wanting to reduce the number of API calls to only those necessary (rather than checking on every run how many long timeout PRs there are).

Reply:
Nope, I was wrong, it seems to be working. The other job was just queued.

Reply:
Let's try the `try`/`catch` approach you suggested, after reading https://github.com/octokit/graphql.js/#errors it should work like we think (`response.errors` wasn't a thing at all)!